routers/context/getRanks.js

```
@@ -1,21 +1,34 @@
 const { UCommon } = require('../../module');
 const { commonParams } = require('../../module/config');
-const dayjs = require('dayjs');
-const isoWeek = require('dayjs/plugin/isoWeek');
-const isoWeekYear = require('dayjs/plugin/isoWeekYear');
-dayjs.extend(isoWeek);
-dayjs.extend(isoWeekYear);
+
+// 缓存 date-fns 模块（避免每次请求都导入）
+let dateFnsCache = null;
+async function getDateFns() {
```

ESLint

[error] Missing space before function parentheses.

(space-before-function-paren)